scripts/gh_ci_summary.py

```diff
@@ -0,0 +1,61 @@
+#!/usr/bin/python3
+
+from datetime import datetime
```

Comment:
Can we add basic unit tests for this code similar to what we have for the Git hooks: https://github.com/timescale/timescaledb/blob/master/scripts/githooks/commit_msg_tests.py

Otherwise, there's a lot of Python code to "just trust" here.